app/controllers/user_confirmations_controller.rb

```diff
@@ -0,0 +1,37 @@
+class UserConfirmationsController < DeviseController
+  include Spree::Core::ControllerHelpers::Auth # Needed for access to current_ability, so we can authorize! actions
+
+  # GET /resource/confirmation/new
```

Comment:
@ltrls My understanding of Devise (it has been a while) is that the devise controllers are used by default unless you specify `controllers` in your `devise_for` call in `routes.rb`. So if you want vanilla devise behaviour I think you should just be  able to omit the `controllers` key altogether.